ops/testing.py

```diff
@@ -521,6 +521,18 @@ def get_workload_version(self) -> str:
         """Read the workload version that was set by the unit."""
         return self._backend._workload_version
 
+    def set_model_info(self, name: str = None, uuid: str = None) -> None:
```

Comment:
You added this method, but didn't add any tests for its behavior.

Reply:
Good point, done